mifosng-provider/src/main/java/org/mifosplatform/portfolio/loanaccount/loanschedule/domain/DefaultScheduledDateGenerator.java

```diff
@@ -62,13 +63,22 @@ public LocalDate adjustRepaymentDate(final LocalDate dueRepaymentPeriodDate, fin
             final LocalDate nextDueRepaymentPeriodDate = getRepaymentPeriodDate(loanApplicationTerms.getRepaymentPeriodFrequencyType(),
                     loanApplicationTerms.getRepaymentEvery(), adjustedDate, loanApplicationTerms.getNthDay(),
                     loanApplicationTerms.getWeekDayType());
-            adjustedDate = WorkingDaysUtil.getOffSetDateIfNonWorkingDay(adjustedDate, nextDueRepaymentPeriodDate,
```

Comment:
Incrementing adjustedDate by 1 day if condition is true.Else we ll call the existing logic getOffSetDateIfNonWorkingDay.